automatic/openvpn/openvpn.nuspec

```diff
@@ -57,6 +57,7 @@ following package parameter can be set:
 * `/DcoDriver`        - install the OpenVPN Data Channel Offload driver
 * `/TapDriver`        - install the TAP-Windows driver (NDIS6)
 * `/WintunDriver`     - install the layer 3 TUN driver for Windows
+* `/PLAP`             - instakl PRE-Logon Access-Provider
```

Comment:
Please correct the spelling on the action instakl --> install and update to use consistent capitalization for PRE - other usage is Pre.